js/src/sdk/client/types.gen.ts

```diff
@@ -1213,7 +2019,7 @@ export type AdvancedUseCaseSearchQueryDTO = {
    * Use case is deprecated. Please provide this in the body instead to avoid max-uri-length error.
    * @deprecated
    */
-  useCase: string;
+  useCase?: string;
```

Comment:
The `useCase` parameter is marked as deprecated in the comment but the type definition doesn't reflect this. Consider:
1. Adding `@deprecated` JSDoc tag
2. Making it optional in the type definition
3. Adding a note about what to use instead